pages/en/release-notes/graphql-validations-migration-guide.mdx

```diff
@@ -0,0 +1,536 @@
+---
+title: GraphQL Validations migration guide
+---
+
+On [deadline date], `graph-node` will support 100% coverage of the [GraphQL Validations specification](https://spec.graphql.org/June2018/#sec-Validation).
```

Comment:
not sure what date to put here yet

Reply:
I'd skip the mention of a deadline for now.